src/main/haskell/kore/src/Kore/Step/Function/Data.hs

```diff
@@ -104,7 +108,11 @@ cases where the function can't be fully evaluated.
 data AttemptedFunction level variable
     = NotApplicable
     | Applied !(OrOfExpandedPattern level variable)
-  deriving (Show, Eq)
+  deriving (Show, Eq, Generic)
+
+instance
```

Comment:
We need to standardize (via the style guide) how to handle multi-line `instance` declarations; there are at least two common ways to break the lines.

Reply:
I fully agree. I don't care what we choose, the default splitting I would pick if nobody else cares is:
```
instance (constraints) => Class Type

instance (long-constraints-that-fit-on-a-line)
    => Class Type

instance
    ( constraints
    , one
    , per
    , line
    ) => Class Type
```

What do you think?

Reply:
:+1: